AutomatticTracks/build.gradle

```diff
@@ -4,6 +4,7 @@ buildscript {
         jcenter()
     }
     dependencies {
+        //noinspection GradleDependency
```

Comment:
Can we make `GradleDependency` issues warnings instead? I think you can do something like this:

```
lintOptions {
        baseline file("lint-baseline.xml")
        warningsAsErrors true
        warning 'GradleDependency'
}
```


or better yet, we can do these in a `lint.xml` file.

Reply:
I'm not sure if I understand - `GradleDependency` [is a lint warning](http://tools.android.com/tips/lint-checks). I just suspended it here as we intentionally don't want to bump Android Gradle Plugin (to have it in sync with other projects as you explained) and I suspended `sentry` bump as there's separate PR for that.

I could disable this warning at all but I believe it is a useful one. What do you think?

Reply:
sorry, I think just understood. You want to override severity once again. So it's warning -> error (`warningAsErrors`) -> warning again (because of `warning`). Cool idea, didn't know it.

I can do that, sure 👍

Reply:
I wasn't able to do this in `lint.xml` file. It looks like this file is interpreted before `lintOptions` extension is configured. This results in having `GradleDependency` as a warning in the moment of evaluating `warningsAsErrors` so eventually it'll be an error.

Maybe I'm missing something, could you make it work with `lint.xml`?

Reply:
I haven't tried it with `lint.xml` either, but I think the order might need to be reversed. In any case, I don't think that should block the PR as the current lint configuration is not that extensive.